sites/dotnet-guide/contents/tutorials/basics/index.md

```diff
@@ -0,0 +1,17 @@
+---
+date: 2020-06-05
+title: The Basics Series
+technologies: [asp.net, .net, csharp]
+topics: [web, data]
+author: khalidabuhakmeh
+subtitle: A set of short basic walkthroughs of different .NET technologies
+thumbnail: ./thumbnail.png
+tutorialItems:
+  - /tutorials/basics/aspnet-mvc/
```

Comment:
Random thought...

Should this one + Razor be part of the ASP.NET series instead?
The current seris can then be "Getting started with Databases in .NET"?

Reply:
On second thought, may make sense to group them like this. Hard :-/